src/app/EventLogging.h

```diff
@@ -73,6 +73,8 @@ CHIP_ERROR LogEvent(const T & aEventData, EndpointId aEndpoint, EventNumber & aE
     eventOptions.mPriority    = aEventData.GetPriorityLevel();
     eventOptions.mFabricIndex = aEventData.GetFabricIndex();
```

Comment:
Wonder if we should check that the event being reported is for a fabric that exists in fabric table

Reply:
would follow-up it, https://github.com/project-chip/connectedhomeip/issues/21982, thanks